x-pack/plugins/cloud_security_posture/public/pages/configurations/findings_flyout/rule_tab.tsx

```diff
@@ -59,58 +61,73 @@ export const getRuleList = (
     title: i18n.translate('xpack.csp.findings.findingsFlyout.ruleTab.tagsTitle', {
       defaultMessage: 'Tags',
     }),
-    description: (
+    description: rule?.tags?.length ? (
       <>
         {rule.tags.map((tag) => (
           <EuiBadge key={tag}>{tag}</EuiBadge>
         ))}
       </>
+    ) : (
+      EMPTY_VALUE
     ),
   },
   {
     title: i18n.translate('xpack.csp.findings.findingsFlyout.ruleTab.frameworkSourcesTitle', {
       defaultMessage: 'Framework Sources',
     }),
-    description: (
-      <CisKubernetesIcons benchmarkId={rule.benchmark.id} benchmarkName={rule.benchmark.name} />
-    ),
+    description:
+      rule?.benchmark?.id && rule?.benchmark?.name ? (
+        <CisKubernetesIcons benchmarkId={rule.benchmark.id} benchmarkName={rule.benchmark.name} />
```

Comment:
not directly related to the PR but it's not `CisKubernetesIcons` anymore for quite a long time already :)